packages/lesswrong/components/review/ReactionsButton.tsx

```diff
@@ -38,21 +38,21 @@ const styles = (theme: ThemeType) => ({
 })
 
 
-const ReactionsButton = ({classes, postId, vote, votes, reaction, freeEntry }: {classes: ClassesType, postId: string, vote: any, votes: ReviewVote[], reaction: string, freeEntry: boolean}) => {
+const ReactionsButton = ({classes, postId, vote, votes, reaction, freeEntry }: {classes: ClassesType, postId: string, vote: any, votes: SyntheticReviewVote[], reaction: string, freeEntry: boolean}) => {
   const voteForCurrentPost = votes.find(vote => vote.postId === postId)
   // TODO: This component is unused, except in ReviewVotingPage2019. Cast to any
   // here is a way to make a minimally invasive fix.
   const currentReactions = (voteForCurrentPost as any)?.reactions || []
   const [freeEntryText, setFreeEntryText] = useState("")
   const [textFieldOpen, setTextFieldOpen] = useState(false)
-  const createClickHandler = (postId: string, reactions: string[], voteId: string | undefined, score: number | undefined) => {
+  const createClickHandler = (postId: string, reactions: string[], score: number | undefined) => {
     if (freeEntry) {
       return () => {
         setTextFieldOpen(true)
       }
     } else {
       return () => {
-        vote({postId, reactions, _id: voteId, previousValue: score})
+        vote({postId, reactions, previousValue: score})
```

Comment:
I wanted to avoid separately querying for votes. This was relatively easy, we already have the information about the vote as a resolver-only field on posts. But once we do so, we lose access to the ID.